recipes/cairo/meson/conanfile.py

```diff
@@ -86,21 +86,21 @@ def layout(self):
         basic_layout(self, src_folder="src")
 
     def requirements(self):
-        self.requires("pixman/0.42.2")
+        self.requires("pixman/0.43.4")
```

Comment:
given expat needs to be bumped to solve a fontconfig conflict, we might as well bump pixman